blueprints/controller-test/qunit-rfc-232-files/tests/unit/__path__/__test__.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('<%= friendlyTestDescription %>', function(hooks) {
+  setupTest(hooks);
```

Comment:
Where do `needs` go?  Should we indicate how those can be added to the registry or is this not a thing anymore?

Reply:
gloriously not a thing anymore.

to stub out lookups (if need be) we use `this.owner.register` -- example:
```js
this.owner.register('service:feature', Service.extend({
    myFeature: false
});
```
probably don't need to document that here though